frontend/packages/knative-plugin/src/components/overview/KnativeTopologyEdgePanel.tsx

```diff
@@ -24,6 +28,8 @@ const connectorTypeToTitle = (type: string): string => {
       return 'Traffic distribution connector';
     case TYPE_EVENT_SOURCE_LINK:
       return 'Event source connector';
+    case TYPE_KAFKA_CONNECTION_LINK:
+      return 'Kafka source connector';
```

Comment:
Technically we can also add support for Strimzi Kafka that have different CR type. Would this be generic Kafka source that can use couple other models or it would be dedicated to KafkaConnection Crs?

I think generic source will be very good and actually simple to add but might be outside scope of the PR.

Reply:
We are not showing `Strimzi Kafka` in the topology. So it is just for `Kafka Source`.